lib/tasks/deployment.rake

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+namespace :deployment do
+  # Run with
+  # be rake "deployment:create_project[project_name]"
+  desc 'Create project'
+  task :create_project, [:project_name] => [] do |_, args| # doesn't need environment
+    @args = args
+    puts "Creating project '#{@args[:project_name]}' - started"
+
+    def create_env_local_file
+      puts 'creating .env.local file...'
+      File.open(File.join(Rails.root, '.env.local'), 'w') do |f|
+        f.puts "PROJECT_NAME:      #{@args[:project_name]}"
+        f.puts "POSTGRES_DATABASE: #{@args[:project_name]}"
+        f.puts "MONGO_DATABASE:    #{@args[:project_name]}"
+      end
+    end
```

Comment:
TODO: only create file if not exists

Reply:
yeah lijken me wel belangrijke checks tbh. Volgens mij kun je dat wel makkelijk checken met `File.exists?` toch?

Reply:
Ow en wat ook fancy zou zijn :-) om hier dezelfde colors als rails gebruikt te gebruiken (bijv bij rails g migration). Maar dat moet je vooral alleen doen als je je verveelt

Reply:
Ik heb het even opgezocht en ze maken gebruik van de een of andere Thor library (https://github.com/erikhuda/thor). Op zich wel hip maar ik kon ook niet hier snel een manier vinden waarop je recursief bestanden uit een subdirectory alleen kopieert als ze nog niet bestaan (dus niet overschrijft). 

Ik heb nu zelf iets geschreven.